lib/destack/src/components/initEditor.tsx

```diff
@@ -0,0 +1,72 @@
+import { loadPanels } from '../lib/panels'
+import { loadTraits } from '../lib/traits'
+import { loadComponents } from '../lib/components'
+import { loadBlocks } from '../lib/blocks'
+import { fetchJSON, escapeName } from '../utils'
+import { appendCss } from '../lib/css'
+import { handleEvents } from '../lib/events'
+
+const initEditor = async ({ server = true }): Promise<void> => {
+  const grapesjs = await import('grapesjs')
+  /*       const uploadFile = (e): void => {
```

Comment:
I notice that `uploadFile` is broken (before this PR). 

Will fix it later and check these changes in depth later when I get to fix it.